### PR TITLE
MahiDebugger Run 3 config

### PR DESCRIPTION
**Pull request by mseidel42**
#### PR description:

MahiDebugger example config, cfi, and plotting script so that it "just runs".

#### PR validation:

Used for an early look at HCAL reconstructed hits from pilot beam test (HCAL not timed in yet): 

image

### CODE REVIEW

**Comment by slava77** [comment]
@cmsbuild please test

File: RecoLocalCalo/HcalRecAlgos/python/mahiDebugger_cfi.py [hunk 1]
```diff
@@ -7,4 +7,5 @@
                                         meanTime = method2.m2Parameters.meanTime,
                                         timeSigmaHPD = method2.m2Parameters.timeSigmaHPD,
                                         timeSigmaSiPM = method2.m2Parameters.timeSigmaSiPM),
-                              mahi.mahiParameters)
+                              mahi.mahiParameters,
+                              recoLabel = cms.InputTag('hbheprereco@cpu'))
```
Comment (slava77) [inline RecoLocalCalo/HcalRecAlgos/python/mahiDebugger_cfi.py:11]: is `@cpu` really needed?

File: RecoLocalCalo/HcalRecAlgos/test/plotMahi.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+            sumEn += thisEnergy
+        if sum(energy) > 0:
+            wTime /= sumEn
+        tex.DrawLatexNDC(0.15, 0.75, 'Mean time = %.1f TS' % (wTime))
+        
+        legend.Draw()
+        
+        c.Print(plotdir+'/'+str(rh.run)+'_'+str(rh.evt)+'_'+str(rh.ieta)+'_'+str(rh.iphi)+'_'+str(rh.depth)+tag+'.pdf')
+        c.Print(plotdir+'/'+str(rh.run)+'_'+str(rh.evt)+'_'+str(rh.ieta)+'_'+str(rh.iphi)+'_'+str(rh.depth)+tag+'.png')
+        count += 1
```
Comment (slava77) [inline RecoLocalCalo/HcalRecAlgos/test/plotMahi.py:104]: add a newline?

**Comment by slava77** [comment]
@mseidel42 
please clarify on the plan to respond to the PR review
https://github.com/cms-sw/cmssw/pull/36143#pullrequestreview-810304245

**Comment by mseidel42** [comment]
Hi Slava, yes, let me do the updates you suggested. I think the @cpu was needed but I'll try it without.

File: RecoLocalCalo/HcalRecAlgos/test/plotMahi.py (hunk 2 above)
Comment (mseidel42) [reply RecoLocalCalo/HcalRecAlgos/test/plotMahi.py]: Fixed

File: RecoLocalCalo/HcalRecAlgos/python/mahiDebugger_cfi.py (hunk 1 above)
Comment (mseidel42) [reply RecoLocalCalo/HcalRecAlgos/python/mahiDebugger_cfi.py]: Fixed, `@cpu` is not needed, indeed!

**Comment by slava77** [comment]
@cmsbuild please test

File: RecoLocalCalo/HcalRecAlgos/python/mahiDebugger_cfi.py [hunk 3]
```diff
@@ -7,4 +7,5 @@
                                         meanTime = method2.m2Parameters.meanTime,
```
[the remark this answers is not in the document]
Comment (mseidel42) [reply RecoLocalCalo/HcalRecAlgos/python/mahiDebugger_cfi.py]: Ah, thank you, I moved it and adapted the `process.load()` accordingly.

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by slava77** [comment]
@cmsbuild please test

it looks like my last request was lost/ignored (I do not see it even in https://cmssdt.cern.ch/jenkins/job/ib-schedule-pr-tests/ )

**Comment by slava77** [comment]
+reconstruction

for https://github.com/cms-sw/cmssw/pull/36143	d9fe520
- code changes are in line with the PR description and the follow up review; there are no changes in the code used in production
- jenkins tests pass

**Comment by perrotta** [comment]
+1
- Test scripts used to inspect HCAL local reco

**Pull request merged**